command/helper/config.go

```diff
@@ -76,6 +77,7 @@ func DefaultConfig() *Config {
 		Consensus:   map[string]interface{}{},
 		LogLevel:    "INFO",
 		RestoreFile: "",
+		BlockTime:   2000,
```

Comment:
Please extract this value `2000` into a constant and reference it, so we can easily change it later